src/ucp/core/ucp_ep.c

```diff
@@ -459,7 +457,8 @@ ucp_ep_sockaddr_server_create_connected(ucp_worker_h worker,
     uct_ep_params_t uct_ep_params;
     char peer_name[UCS_SOCKADDR_STRING_LEN];
     ucs_status_t status;
-    ucp_md_index_t i;
+
+    UCS_STATIC_ASSERT(UCS_SOCKADDR_STRING_LEN <= UCP_WORKER_NAME_MAX);
```

Comment:
why need this? just use UCP_WORKER_NAME_MAX in UCP, instead of UCS_SOCKADDR_STRING_LEN

Reply:
I increased `UCP_WORKER_NAME_MAX`, this check is needed to make sure that `peer_name` and `ep->peer_name` have enough space. `ucs_sockaddr_str` may cut the name without this check if use `UCP_WORKER_NAME_MAX` everywhere.